### PR TITLE
#48: refactoring

### PR DESCRIPTION
**Pull request by ggajos**
for #48

### CODE REVIEW

**Comment by karato** [comment]
Thanks, I will ask someone to do a code review here, thanks for your pull request

**Comment by karato** [comment]
@krzyk review it

File: src/test/java/com/nerodesk/TkIndexTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * Tests for {@code TkIndex}.
+ *
+ * @author Grzegorz Gajos (grzegorz.gajos@opentangerine.com)
+ * @version $Id$
+ * @since 0.2
+ */
+public final class TkIndexTest {
+
+    /**
+     * TkIndex can return alive note.
```
Comment (krzyk) [inline src/test/java/com/nerodesk/TkIndexTest.java]: The name of test and it's description are different.
Maybe description should be: _TkIndex can return plan text_?

File: src/test/java/com/nerodesk/TkIndexTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+     *
```
Comment (krzyk) [inline src/test/java/com/nerodesk/TkIndexTest.java]: I think it will look better without the additional new line between description and `@throws`

**Comment by krzyk** [comment]
@ggajos two comments from me

**Comment by ggajos** [comment]
@krzyk thanks for feedback, please have a look if everything is fine now

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by ggajos** [comment]
@rultor merge please

**Comment by karato** [comment]
@krzyk Thank you, I have added **17 mins** to your account in payment/transaction "`52967478`"

extra minutes for review comments (c=2)

added +17 to your rating, now it is equal to [+4919](http://www.netbout.com/b/35035?open=rating)

**Comment by karato** [comment]
@rultor deploy now pls

**Pull request merged**